pkgs/top-level/python-packages.nix

```diff
@@ -24571,6 +24571,22 @@ in {
     };
   });
 
+  sphinxcontrib_newsfeed = buildPythonPackage (rec {
+    name = "sphinxcontrib-newsfeed-${version}";
+    version = "0.1.4";
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/2b/5e/8bc839b5c4ef030bf26eede24208a49f25d00033cbd4969b3895264f14db/${name}.tar.gz";
```

Comment:
You could prefer

```
url = "mirror://pypi/s/sphinxcontrib-newsfeed/${name}.tar.gz";
```

With this form, future updates are simpler (`version` and `sha256` have to be updated, not the `url`).